### PR TITLE
feat: 🌐 Add 🇺🇦 Ukrainian translation

### PR DESCRIPTION
**Pull request by kopach**
Ref: https://github.com/rxhanson/Rectangle/discussions/1094

### CODE REVIEW

File: Rectangle/uk.lproj/Main.strings [hunk 1]
```diff
@@ -1,550 +1,550 @@
...
+"Left half, top/bottom half near corners" = "Ліва половина, верхня/нижня половина близько кутів";
+"Right half, top/bottom half near corners" = "Права половина, верхня/нижня половина близько кутів";
+"Thirds, drag toward center for two thirds" = "Третини, перетягніть у центр для двох третин";
+"Thirds, top/bottom half near corners" = "Третини, верхня/нижня половина близько кутів";
+"Left or right half" = "Ліва або права половина";
+"Top sixths from corners; maximize" = "Верхні шості від кутів; максимізувати";
+"Bottom sixths from corners; thirds" = "Нижні шості від кутів; третини";
+"Fourths columns" = "Четвертини стовпців";
+
+"Go to System Settings → Privacy & Security → Accessibility" =
```
Comment (rxhanson) [inline Rectangle/uk.lproj/Main.strings:546]: Everything else looks good, just this have this missing one.

**Review by rxhanson — CHANGES_REQUESTED** [review]
Thanks! Just one missing one near the end.

File: Rectangle/uk.lproj/Main.strings (hunk 1 above)
Comment (kopach) [reply Rectangle/uk.lproj/Main.strings]: oh, sorry, missed that somehow. Fixed now

**Comment by kopach** [comment]
> Thanks! Just one missing one near the end.

fixed @rxhanson. Please, take a look once again, thanks

Btw, how about this file https://github.com/rxhanson/Rectangle/blob/aa434500ec61292c5bf7da92c8fb0de477abdf54/RectangleLauncher/uk.lproj/Main.strings 
Should it also be translated?

**Comment by rxhanson** [comment]
Thanks! That file doesn't need to be translated (it's the embedded helper app without a UI that is used to launch the app on startup for macOS < 13).

**Review by rxhanson — APPROVED** [review]

**Pull request merged**